public/styles.css

```diff
@@ -186,4 +193,26 @@ ul {
 
 .publisherCol {
   padding-top: 4px;
+}
+
+.overlay {
+  margin: 0px;
+  padding: 5px;
+  word-wrap: normal;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  color: rgb(51, 51, 51);
+  z-index: 1;
+  position: absolute;
+  right: 5px;
+  top: 207px;
+  border: 1px black solid;
+  border-radius: 3px;
+  background-color: rgb(161, 162, 163);
+  box-shadow: 0.5px 0.5px 2px;
+  width: 310px;
+}
+
+.reviewStats {
+  width: 300px;
 }
```

Comment:
Try experimenting with adaptive units of measurements such as em, rem and vw and vhs.

Reply:
In the past I have. The only reason I've been using px is because everything in the actual app is set to very specific dimensions. I'll keep it in mind though!